### DISCUSSION TITLE
Lazy-load components in Vue 3

### DISCUSSION THREAD

**Discussion by aabm00** [quasarframework/quasar#9324] [General - Components / Directives / etc]
Hi !

To lazy load components, in the docs say :  

`<script>
export default {
  components: {
    SomeComponent: () => import('components/SomeComponent'),
  }
}
</script>`

it works in Vue 2 but it didn't work  for me in Vue 3

It had to be: 
  `SomeComponent: (() => import('components/SomeComponent')),`

Only for if someone had the same issue

**Comment by metalsadman** [comment]
@aabm00 https://v3.vuejs.org/guide/migration/async-components.html#_3-x-syntax
